src/app/style/module/ellipsis-inline/ellipsis-inline.scss

```diff
@@ -2,4 +2,5 @@
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  font-size: 18px;
```

Comment:
Si, avevo notato. Per questo ho inserito qui l'attributo font-size, in modo da rendere uniforme la dimensione del testo nelle card dei giocatori salvati con quella del testo della card "aggiungi giocatore".